### PR TITLE
Removes big manipulator's use mode

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Removes big manipulator's use mode due to how incredibly janky and unstable it is in the background. We should not be spawning a human every time an item is used.

## Why It's Good For The Game

This can cause extreme amounts of lag and almost crashed manny recently.

## Changelog
:cl:
del: Removed big manipulator's use mode
/:cl:

### CODE REVIEW

**Comment by IndieanaJones** [comment]
A genuine case of ghost in the machine, except there's hundreds or even thousands of ghosts and they still have organs under the right conditions

**Comment by MrMelbert** [comment]
If anyone wants to re-add this, I think it'd work fine if the manipulator itself was just a stationary basic mob. Though you'd have to work around a few memes (mind transfer pops out to me)

File: code/game/machinery/big_manipulator.dm [hunk 1]
```diff
@@ -297,61 +291,6 @@
...
-		target.forceMove(drop_turf) /// We use only items
-		target.dir = get_dir(get_turf(target), get_turf(src))
-		finish_manipulation()
-		return
-	var/obj/item/im_item = target
-	var/atom/type_to_use = search_type_by_priority_in_drop_turf(allowed_priority_settings)
-	if(isnull(type_to_use))
-		check_end_of_use(im_item, target, item_was_used = FALSE)
-		return
-	var/mob/living/carbon/human/dummy/living_manipulator_lmfao = create_abstract_living()
```
Comment (Iamgoofball) [inline code/game/machinery/big_manipulator.dm:325]: hahahaha what the fuck is this code

**Comment by zxaber** [comment]
> If anyone wants to re-add this, I think it'd work fine if the manipulator itself was just a stationary basic mob. Though you'd have to work around a few memes (mind transfer pops out to me)

Could just keep a generic mob in its own contents and hand off any item use procs to it. I tried to do this feature years ago and that's how I handled it. No repeatedly making a new mob; just spawn one during init and qdel it during destroy.

**Comment by SyncIt21** [comment]
Wouldn't it be best to test merge till someone fixes it instead of full merge?

**Comment by IndieanaJones** [comment]
Would probably be best to full merge because who knows how long it'll take for someone to code the feature.

**Comment by Time-Green** [comment]
> Wouldn't it be best to test merge till someone fixes it instead of full merge?

I don’t want to rush them. This way they can take their time to implement it properly

**Comment by dwasint** [comment]
> > If anyone wants to re-add this, I think it'd work fine if the manipulator itself was just a stationary basic mob. Though you'd have to work around a few memes (mind transfer pops out to me)
> 
> Could just keep a generic mob in its own contents and hand off any item use procs to it. I tried to do this feature years ago and that's how I handled it. No repeatedly making a new mob; just spawn one during init and qdel it during destroy.

all fine and good until some obscure interaction between mobs happens or someone converts the manipulator into a cultist.

**Pull request merged**